driver/ppm_flag_helpers.h

```diff
@@ -53,11 +56,13 @@ static __always_inline uint32_t open_flags_to_scap(unsigned long flags)
 	if (flags & O_EXCL)
 		res |= PPM_O_EXCL;
 
+#ifndef WDIG
 	if (flags & O_NONBLOCK)
 		res |= PPM_O_NONBLOCK;
 
 	if (flags & O_SYNC)
 		res |= PPM_O_SYNC;
+#endif 
```

Comment:
Do we want these wrapped in `#ifdef O_NONBLOCK` and `#ifdef O_SYNC`, respectively?

Reply:
fixed